README.md

```diff
@@ -55,9 +58,24 @@ models:
     +schema: your_destination_schema # optional, default is your target database
     staging:
       +schema: your_destination_schema # optional, default is your target schema
+  ...
 ```
 
-Note that the model materializations are defined in this package's `dbt_project.yml`, so do not set them in your project.
+You can also implement your own logic by overriding the default [generate_database_name](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-databases)
+and [generate_schema_name](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-schemas) macros.
+
+Note that the model materializations are defined in this package's `dbt_project.yml` as views. Incremental logic is set up in the
+staging models and if you want to leverage this materialization, you need to specify the following configuration in the `dbt_project.yml`
+of your project:
+
+```yml
+models:
+  ...
+  dbt_artifacts:
+    staging:
+      +materialized: incremental
+  ...
+```
```

Comment:
Let's remove this as it's not within scope of the PR if ok :)

Reply:
sure ;) and will put it in a separate PR.

Reply:
@NiallRees : removed from the PR